CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
-
-See a list of the available documentation versions in the
-[branches page](https://github.com/knative/docs/branches) of the `knative/docs`
-repo.
-
-## Assigning owners and reviewers
-
-For both documentation and code samples, you should assign your PR to a single
-owner ("_Assignee_") using the `/assign` Prow command. It's best to set the
-"Assignee" and include other stakeholders as "Reviewers" rather than leaving it
-unassigned or allowing [Prow](https://prow.k8s.io/command-help) to auto assign
-reviewers.
-
-Use the `/assign` command to set the owner. For example: `/assign @owner_id`
-
-For code samples, initially set the owner of your PR to the SME who should
-review for technical accuracy. If you don't know who the appropriate owner is,
-nor who your reviewers should be for your PR, you can assign the
-[current working group lead](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md)
-of the related component.
-
-If you want to notify and include other stakeholders in your PR review, use the
-`/cc` command. For example: `/cc @stakeholder_id1 @stakeholder_id2`
-
-## Docs contributor roles
-
-Because contributing to the documentation requires a different skill set than
-contributing to the Knative code base, we've defined the roles of documentation
-contributors separately from the roles of code contributors.
-
-If you're looking for code contributor roles, see
-[ROLES](https://github.com/knative/community/blob/master/ROLES.md).
-
-### Member
-
-Established contributor to the Knative docs.
+The
+[Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets biweekly on Tuesdays at 9:30am PT.
+[Click here](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar.
 
-Members are continuously active contributors in the community. They can have
-issues and PRs assigned to them, might participate in working group meetings,
-and pre-submit tests are automatically run for their PRs. Members are expected
-to remain active contributors to the Knative documentation.
+The Working Group meetings are used to discuss documentation strategy and
+policy; we expect individual PR review to happen mostly asynchronously via
+GitHub, and [Slack](https://slack.knative.dev).
 
-All members are encouraged to help with PR reviews, although each PR must be
-reviewed by an official [Approver](#approver). In their review, members should
-be looking for technical correctness of the documentation, adherence to the
-[style guide](https://developers.google.com/style/), good spelling and grammar
-(writing quality), intuitive organization, and strong documentation usability.
-Members should be proficient in at least one of these review areas.
+# Workflow
 
-### Requirements
+There are roughly two workflows: writing / contributing new content, and
```

Comment:
I think we need to decouple the tech writing process from the process of engineers getting content into the docs repo. My thought is:

10 PRs come in; 6 are "new content", 4 are "refine content".
* 3 "new content" PRs are serving --> Serving/Networking reviewers verify content
* 2 are "install" --> operations reviewers verify content
* 1 is a sample --> UX reviewers verify content
* 4 refine content --> SMEs review content, with tech writers as backstop if SME is not responsive

This would hopefully reduce the amount of toil that tech writers need to do, and move the burden of getting content into the repo onto the various teams producing the changes, so they can move the content in at their own rate.